### PR TITLE
Update monero-core_hr.ts

### PR DESCRIPTION
**Pull request by Fajl**
added missing translation for Croatian language

### CODE REVIEW

File: translations/monero-core_hr.ts [hunk 1]
```diff
@@ -1220,155 +1220,155 @@ Datoteka sa starom predmemorijom novčanika biti će preimenovana i može se nak
     <message>
         <location filename="../pages/SharedRingDB.qml" line="93"/>
         <source>Shared RingDB</source>
-        <translation type="unfinished"></translation>
+        <translation">Zajednička baza podataka o prstenovima</translation>
```
Comment (erciccione) [inline translations/monero-core_hr.ts]: remove `"` inside `<translation>`

File: translations/monero-core_hr.ts [hunk 2]
```diff
@@ -1220,155 +1220,155 @@ Datoteka sa starom predmemorijom novčanika biti će preimenovana i može se nak
...
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="134"/>
         <source>This sets which outputs are known to be spent, and thus not to be used as privacy placeholders in ring signatures. </source>
-        <translation type="unfinished"></translation>
+        <translation">Ovdje se unosi popis poznatih potrošenih izlaza kako se nebi ponovo koristili u potpisima prstenova.</translation>
```
Comment (erciccione) [inline translations/monero-core_hr.ts]: Again, remove `"` inside `<translation>`

**Comment by erciccione** [comment]
Yes, al good. Now please squash those commits in just one, if you don't know how to it, i can help you.

**Comment by Fajl** [comment]
i followed an online guide to squashing commits, hopefully everything is okay!

**Comment by erciccione** [comment]
Actually no, you now have 3 commits + a merge commit. I don't think you can do that using github, you should use git locally. If you want to learn how to do it i can teach you, if not, i can just squash for you.

**Comment by erciccione** [comment]
ok, so please add me as contributor to your repository, so i can make the changes for you.

**Comment by Fajl** [comment]
Double checked everything, looks good to me. Thanks for your help! Have a nice day

**Pull request merged**